crawl-ref/source/dat/strings/de/adjectives-e.txt

```diff
@@ -738,7 +738,7 @@
 "feurige "
 %%%%
 "obsidian "
-"Obsidian-"
+"vulkanische "
```

Comment:
This is ok, but I feel like we're losing some specificity here. I mean, they could've called it "the volcanic rune" in English, but they didn't.

Reply:
Also a case of trying to get rid of the hyphen. My reasoning was that obsidian is created from lava. 'Obsidianrune' would be perfect, if that's an option?

Reply:
Yeah. We can do "Obsidianrune" (see my response in relation to Spinnfadenrune).

Reply:
Excellent!